Resources/Prototypes/_White/Recipes/hidden_crafts.yml

```
@@ -0,0 +1,11 @@
+- type: constructionGraph
+  id: EnergyDoubleSwordGraph
+  start: esword
+  graph:
+  - node: esword
+    edges:
+      - to: desword
+        steps:
+          - tag: EnergySword
+  - node: desword
+    entity: EnergySwordDouble
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)